scripts/start_lisk

```diff
@@ -55,22 +53,22 @@ populate_database() {
 
   if [ "$DATABASE_HOST" == "localhost" ]
   then
-    echo "Looking for database ${DATABASE_NAME}_check"
-    psql -ltAq | grep "^${DATABASE_NAME}_check|"
+    echo "Looking for peers table"
+    psql -qd "$DATABASE_NAME" -c "select count(*) from peers;" &> /dev/null
```

Comment:
I am not so sure if i like this implementation. In the future peers table may go away with nodes always rediscovering peers when they come online.